### ISSUE TITLE
[processor/metrictransform] Aggregating labels drops metric description

### ISSUE THREAD

**Issue by BinaryFissionGames** [open-telemetry/opentelemetry-collector-contrib#14577]
## Description
When aggregating by labels, the metric description is made empty.

See this function:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d5bb86ce9ec34a62a6959ed22cc5bd61a6611215/processor/metricstransformprocessor/metrics_transform_processor_otlp.go#L454-L470

The metric description is not copied over to the new metric, which causes the description of the metric to be dropped.

## Steps to Reproduce
Run with attached config.
Observe output metric descriptions are empty.

## Expected Result
Metric description is retained on the metric with aggregated labels

## Actual Result
Metric description is dropped on the metric with aggregated labels

### Collector version

v0.60.0-47-g0bf2d8429d

### OpenTelemetry Collector configuration

```yaml
receivers:
  hostmetrics:
    collection_interval: 5s
    scrapers:
      filesystem:

processors:
    metricstransform:
      transforms:
        # Keep only direction/request metrics
        - include: ^system.filesystem
          match_type: regexp
          action: update
          operations:
            - action: aggregate_labels
              label_set: [ device, mode, mountpoint, type]
              aggregation_type: sum

exporters:
  logging:
    loglevel: debug

service:
  pipelines:
    metrics:
      receivers: [hostmetrics]
      processors: [metricstransform]
      exporters: [logging]
```

**Comment by abhide** [comment]
@dmitryax I would like to work on this issue. Can you please assign it to me?

**Comment by dmitryax** [comment]
Thanks, @abhide. Assigned to you

**Comment by BinaryFissionGames** [comment]
Closing as resolved by #20442 (Thanks @abhide!)
